src/aioswitcher/bridge.py

```diff
@@ -173,13 +173,15 @@ class SwitcherBridge:
     def __init__(
         self,
         on_device: Callable[[SwitcherBase], Any],
-        broadcast_ports: list[int] = [SWITCHER_UDP_PORT_TYPE1, SWITCHER_UDP_PORT_TYPE2],
+        broadcast_ports: List[int] | None = None,
```

Comment:
This changes the functionality of the code and will break existing implementation
```suggestion
        broadcast_ports: List[int] | None = [SWITCHER_UDP_PORT_TYPE1, SWITCHER_UDP_PORT_TYPE2],
```

Reply:
@thecode good catch!
that was actually suggested by the _sonarlint_ extension,
I must have committed this by accident, my bad.

I think your suggestion was what _sonarlint_ complained about to begin with,
let me see if I can work something out that will both satisfy _sonarlint_ and not break anything.

That being said, the tests pass on my computer,
so I guess we don't test this potential breaking point very well.

(I don't think the CI failures are related to , the same tests pass on my computer.
so I guess it's missing requirement or something, but I haven't had the change to work on this yet).

Reply:
![image](https://user-images.githubusercontent.com/28388442/192137914-57cf0cf3-b00b-4b75-863b-85e43fee59b3.png)

Reply:
@thecode 
fixed here https://github.com/TomerFi/aioswitcher/pull/559/commits/0647e41f333a2557fa92d7888d98f0a753090828, please take a look.

Reply:
> > As I wrote on Discord, I think we should make our typing Python 3.10 compatible and avoid need to change it in the future
> 
> and as I explained in Discord, we're still officially supporting 3.9... and this project is fully 3.10 compatible, I think you are mistaken.

Check other projects and see that they align code with the most recent supported version and not the older one, they just check backward compatibility vs oldest supported version. It looks weird to make a PR to align code to older version.
However it is your repo, feel free to merge but IMO at least the comment about `broadcast_ports` should be fixed, it is not related to a Python version (I am doing it on projects that are still Python 3.7 compatible with typing.